src/python/Utils/Utilities.py

```diff
@@ -299,3 +295,5 @@ def encodeUnicodeToBytesConditional(value, errors="ignore", condition=True):
     if condition:
         return encodeUnicodeToBytes(value, errors)
     return value
+
+makeList("")
```

Comment:
I guess this line shouldn't be here(?)

Reply:
yes, you are correct, I added it to debug and then forgot to delete :/